### DISCUSSION TITLE
Can I store full file path in fileUpload

### DISCUSSION THREAD

**Discussion by ayemin-tun** [filamentphp/filament#10712] [General]
### Description
Im my filament I use 

```
 FileUpload::make('image')
                        ->required()
                        ->image()
                        ->imageEditor()
                        ->columnSpanFull(),
```

  and but it is store img name only so when I call this from frontend like next.js the image field does not show image because it only image name eg -> 01HKPAVVS21PEMGX8PNV20DK5V.png 

  I want to store full image path in image like http://dev.tennis.fed.com/storage/01HKPAVVS21PEMGX8PNV20DK5V.png

**Comment by ShaungBhone** [comment]
I don't think you don't need to store the full path. You can retrieve the image for next.js by using Laravel ways. https://laravel.com/docs/10.x/filesystem#retrieving-files.

**Comment by devsumitkarn** [comment]
DALL·E 2025-02-15 00.35.52 - A vibrant outdoor nature concert scene surrounded by lush green forests and colorful flowers. The stage is beautifully decorated with string lights, a (1)

i was this path uploading but 

i can upload this in filament
